src/gui/wizard/owncloudadvancedsetuppage.cpp

```diff
@@ -344,6 +432,17 @@ void OwncloudAdvancedSetupPage::slotSelectFolder()
     setErrorString(errorStr);
 }
 
+
+void OwncloudAdvancedSetupPage::setLocalFolderPushButtonPath(const QString &path)
+{
+    auto prettyPath = path;
+
+    const QString homeDir = QDir::homePath() + QString("/");
```

Comment:
Better use `QLatin1Char('/')` this should make for a more efficient concat and avoid a couple of allocations. Also are we sure `homePath()` never carries the final '/'? (I confess I always have to look that up...)

Reply:
OKay. No I'm not sure. But in during my local tests it never had one. Regardless I have no added a check.

Reply:
After checking the doc it could indeed return "/" or "C:/" as fallback. Admittedly a rare case but better safe than sorry.